learntools/ml-insights/nbs/tut3.ipynb

```diff
@@ -0,0 +1,215 @@
gAyADIAMgDWsYu5fa4LgHNkAGQAZABkAGQAkWWAxi4cba4LgHNk\nAGQAZABkAGQAkWWAxi4cY1wXAOfIAMgAyADIAMgAIssAjV04NrsuAM6RAZABkAGQAZABRJYBGrtw\njHddAJwjAyADIAMgAyADiCwDNHbh4HUFGQAZABkAGQAZQGQZYB27EBhjKnzfb3RdB9whAyADIAMg\nAyADiDID/BUhHFNcFwDnyADIAMgAyADIACLLAI1dOOpcFwDnyADIAMgAyADIACLLAI0dAAAAAMQc\njV04RrkuAM6RAZABkAGQAZABRJYBLp4SAj4oCzIAMgAyADIAMoAoM1DSnwclEokiSd+WNF9Si6Sr\n0+n0mi72+YOk+9Lp9G1HWmjMTJL0musi4BQZABkAGQAZABlAZBno71TMyySVp9PpsyV9VtLXutjn\ni5JG97ewmOtwXQCcIwMgAyADIAMgA4gsA/1t7BZJekCS0un0U5JOy92YSCTeK3sSDxxRdfG13XUB\ncI4MgAyADIAMgAwgsgz0ayqmpGpJe3N+bk8kEiXpdLotkUicIOkKSe+VdHNPB7nzzjuvkXSNJNXV\n1d19ww03/FHSVkljJZVJ2iBpmqR9ktokjZG0WdJ42aZ0s+zaEJ2XER0laaPskGeH7As5SdKe4Fyr\nc47ZKmm3pImSdkkql1SVs705OO4ESTslVUqqyNneKKlBUo2kbcFzlwfbFxljXgyOMW6QnNM0SfWc\nU6/PqdoY0zjIzmkwvk9hntOE4L8Dg+mcBuP7FOY5LTLGPDfIzmkwvk9hntM4Y0z9IDunwfg+hXlO\nNcaY1YPsnAbj+xTmOZ1jjFnZ23PyfX+1+qlfF09JJBJfl/RUOp2+N/h5Uzqdnhx8/xVJ50pqkjQ9\nOKnr0un0kBm9M8bU+L6/03UdcIcMgAyADIAMgAwgygz0d8TucUmXSro3kUicJenFzg3pdPqfO79P\nJBKfl7RtKDV1gf6+rhg8yADIAMgAyADIACLLQH8/Y/cbSc2JROIJSf8t6fpEInFDIpF458CVFmvV\nrguAc2QAZABkAGQAZACRZYB17EJgjCn3fb/ZdR1whwyADIAMgAyADCDKDPR3xA49m+a6ADhHBkAG\nQAZABkAGEFkGaOzC0eq6ADhHBkAGQAZABkAGEFkGaOzCsdt1AXCODIAMgAyADIAMILIM0NiFY6Lr\nAuAcGQAZABkAGQAZQGQZoLELxy7XBcA5MgAyADIAMgAygMgyQGMXjnLXBcA5MgAyADIAMgAygMgy\nQGMXjirXBcA5MgAyADIAMgAygMgywDp2IWDNEpABkAGQAZABkAGwjl38sWYJyADIAMgAyADIAFjH\nLub4ywzIAMgAyADIAMgAIssAjV046lwXAOfIAMgAyADIAMgAIssAjV04JrguAM6RAZABkAGQAZAB\nRJYBGrtw7HRdAJwjAyADIAMgAyADiCwDNHbhqHRdAJwjAyADIAMgAyADiCwDNHbhqHBdAJwjAyAD\nIAMgAyADiCwDrGMXAtYsARkAGQAZABkAGQDr2MUfa5aADIAMgAyADIAMgHXsYq7RdQFwjgyADIAM\ngAyADCCyDNDYhaPBdQFwjgyADIAMgAyADCCyDNDYhaPGdQFwjgyADIAMgAyADCCyDNDYhWOb6wLg\nHBkAGQAZABkAGUBkGaCxC8co1wXAOTIAMgAyADIAMoDIMkBjF45y1wXAOTLQC8lMtiyZyQ7W/w6R\nAZABkAGQAUSWAdaxCwFrloAMHBQ0buU5t2pJIyWNkFQsqV3SVkk7JNXdvnjOAUelDigyADIAMgAy\ngCgzQGMXAmPMbN/3V7uuA+4MxQwkM9lSHWzeKmSbt+rgexPs5ks6IKk1uHXIzhyokjQs2GeXpM2S\n9t6+eE5sLxM9FDOAQ5EBkAGQAUSZgZIonmQIqnddAJwblBkIRt+G6dDRt2rZ0bcy2cZNsg1bq6QW\nSbsPc9gOSftyfq6QdJIkk8xk62WbvD2S9t++eE7HwJxJJAZlBtAnZABkAGQAkWWAEbsQGGPG+r5/\nuF9mMYjFPQPJTLZEB5u34To4+lalg6Nvkh19awm+todQSpls01gk2yh2Ttnce/viOW0hPN+AiXsG\ncOTIAMgAyACizAAjduEYp8OPUmBwK/gMJDNZo0NH30bo4AjcMB0cffNlm7dWRX9Ouc9ZLGmipGmS\nOpKZ7C5JWyS1BTXm1tvdz73Zp8vH3L54Tl//ClbwGUDoyADIAMgAIssAI3YhMMZU+76/7/B7YrAq\npAwkM9liHTr6lnvxktzRtzYdbODCGH0baJWy5+Pr0PPI/7kr/XlM536+7PTRDh069bRNduSyTVLr\nxuWZsilnLq7Vwc8Stvdw6+hH44gCV0j/HYAbZABkAFFmgBG7cIzVoZ8ZwtATaQaC0bcyHWzgqnRw\n+mS5DjYg0sHmbU/e/XHTENyiZnK+mryfO9+Doqa6PZNkp40Wd3EMP+cxkqRkJtvZFHZeXCb/a5u6\naQpzbzH7HOJgx/8LQAZABhBZBmjswlHmugA4F3oGkplspaQpOnTpgE5tss1Ak6T9YdcyxPh5X3P9\ndamGhh3bmyXV9uG4RcGtWHYksvPzjMXB/blNZH49uQ2idHAEsT34mvtze9623PtzRyO7ur1pOyON\nPeL/BSADIAOILANMxQwBa5Yg7AwETd2Zsr/QN8uOwvGPuYDs3fRG2cjJU1sdltDZKJq8r0U9bO/0\nphHFPLnbfXXdMObfl99I9tgwdrU9bqOR/L8AZABkAKxjF3OsWYIwM5DMZKsknSH7C7KLqYjohed+\n/N1pCz58zQbXdUTkcA1jV9u7criGUnpzs3i4hrJzxLJXI5C52450NJL/F4AMgAyAdezij7nUCCUD\nNHXxMWLC0UPp/elshqKQ3zCWBLeeGsrejEC+aVsyk82fwpo/tTW/oez86kvqm
```

Comment:
I wonder if there's a better example that could be used here. I wasn't able to get an intuitive takeaway about this pair of variables.